swath_projector/exceptions.py

```diff
@@ -52,3 +52,15 @@ def __init__(self):
         super().__init__(
             'InvalidTargetGrid', 'Insufficient or invalid target grid parameters.'
         )
+
+
+class VariableShapeError(CustomError):
```

Comment:
Nice catch. This exception is not being used anywhere. It was created by Matt during his initial implementation of the quick fixes but since it didn't end up being used, I will remove it.

Reply:
Fixed in commit [735894d](https://github.com/nasa/harmony-swath-projector/pull/19/commits/735894d1f7c3771c9aa53047e64669af8db5f877)